### DISCUSSION TITLE
Use Specific Self-Hosted Action Runner?

### DISCUSSION THREAD

**Discussion by vinaysandela** [community/community#25716] [Actions]
<p>We got multiple self-hosted Action Runners with similar Tag.<br>
Want to use a specific Self-Hosted Action Runner. How can I mention that on my workflow ?</p>
<p>Azure DevOps - We used Agent.Name equals AgentName<br>
Looking for similar actions.</p>

**Comment by jsoref** [comment]
<p>Is this not good enough?</p>
<pre><code class="lang-auto">runs-on: [self-hosted, some-label]
</code></pre>
<p>(Each runner can have a unique label in addition to common labels.)</p>

**Comment by mishmanners** [comment, marked as answer]
<p>You can use self-hosted runners by following the GitHub Docs: <a href="https://docs.github.com/en/actions/hosting-your-own-runners/using-labels-with-self-hosted-runners" class="inline-onebox">Using labels with self-hosted runners - GitHub Docs</a></p>
<p>There are labels you can use.</p>
